testutil/compose/lock.go

```diff
@@ -66,20 +66,24 @@ func Lock(ctx context.Context, dir string, conf Config) (TmplData, error) {
 
 		var nodes []TmplNode
 		for i := 0; i < conf.NumNodes; i++ {
-			n := TmplNode{EnvVars: newNodeEnvs(i, conf, "")}
+			n := TmplNode{
+				EnvVars:    newNodeEnvs(i, conf, ""),
+				Entrypoint: "sh",
+				Command:    fmt.Sprintf("[-c,'%s %s && sleep 2']", conf.entrypoint(), cmdDKG), // Sleep after completion to allow other nodes to finish
```

Comment:
```suggestion
				Command:    fmt.Sprintf("[-c,'%s %s && sleep 2']", conf.entrypoint(), cmdDKG), // Sleep for 2s after completion to allow other nodes to finish
```

Reply:
I prefer not to add details like that so that the risk of the comment becoming out of data/out of sync is lower.

And remember: comments should say why the code does something, not what the is doing.